src/charm_state.py

```diff
@@ -58,6 +58,7 @@ class SynapseConfig(BaseModel):  # pylint: disable=too-few-public-methods
 
     Attributes:
         allow_public_rooms_over_federation: allow_public_rooms_over_federation config.
+        enable_irc_bridge: enable_irc_bridge config.
```

Comment:
Perhaps describe what it is intended to enable? I'm not sure the description provides any more information?

Reply:
Done.